tests/TestableIO.System.IO.Abstractions.TestingHelpers.Tests/MockFileSystemEventsTests.cs

```diff
@@ -0,0 +1,840 @@
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.Serialization;
+using System.Runtime.Serialization.Formatters.Binary;
+using System.Threading;
+using System.Threading.Tasks;
+using NUnit.Framework;
+using System.IO.Abstractions.TestingHelpers.Events;
+
+namespace System.IO.Abstractions.TestingHelpers.Tests;
+
+using XFS = MockUnixSupport;
+
+public class MockFileSystemEventsTests
+{
+    [Test]
+    public void Events_WhenNotEnabled_ShouldNotFireEvents()
+    {
+        var fileSystem = new MockFileSystem();
+        var eventFired = false;
+        
+        // Events exist but are not enabled
+        Assert.That(fileSystem.Events, Is.Not.Null);
+        Assert.That(fileSystem.Events.IsEnabled, Is.False);
+        
+        // Subscribe should work but events won't fire
+        using (fileSystem.Events.Subscribe(args => eventFired = true))
+        {
+            fileSystem.File.Create(XFS.Path(@"C:\test.txt")).Dispose();
+            Assert.That(eventFired, Is.False);
```

Comment:
General: Please replace the NUnit assertions with [aweXpect](https://awexpect.com/) to be consistent with the other test classes.